src/payment/models/crypto-input/btc-input.service.ts

```diff
@@ -135,7 +137,11 @@ export class BtcInputService extends CryptoInputService {
   }
 
   private async forwardUtxo(input: CryptoInput, address: string): Promise<void> {
-    const outTxId = await this.btcClient.send(address, input.inTxId, input.amount, input.vout);
+    const { fastestFee } = await this.callApi<{ fastestFee: number; halfHourFee: number; hourFee: number }>(
+      this.btcFeeUrl,
+    );
+    const btcFee = fastestFee < Config.crypto.fee * input.amount ? fastestFee : Config.crypto.fee * input.amount;
```

Comment:
I think this is not related to `Config.crypto.fee`, which is what the user pays. The factor here should be `0.0005% = 0.000005`, the user fee will be much higher.

Code improvement: `const btcFee = Math.min(fastestFee, 0.000005 * input.amount);`

Reply:
Move the logic to separate method `getFeeRate()` or similar.